### PR TITLE
feat(ui5-menu): add accessibleName property

### PR DESCRIPTION
**Pull request by tsanislavgatev**
fixes: https://github.com/SAP/ui5-webcomponents/issues/5814

### CODE REVIEW

File: packages/main/src/Menu.js [hunk 1]
```diff
@@ -44,6 +44,18 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 * Defines the accessible ARIA name of the component.
+		 *
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 * @since 1.9.0
```
Comment (ilhan007) [inline packages/main/src/Menu.js:53]: 1.9.0 is released already

**Comment by ilhan007** [comment]
Edit the "Menu acc name" to  follow the commit message guidelines

**Comment by ilhan007** [comment]
The issue that is addressed is not referenced.

**Comment by tsanislavgatev** [comment]
> The issue that is addressed is not referenced.

Yes, for both of the comments, they are not implemented, because we still need to discuss with ACC expert.

**Review by unazko — APPROVED** [review]
From what we've tested the same issue is also reproducible for other ui5 components which open a popover. The accessible-name API isn't used in the ui5-date-picker for example as well.
According to the specification we are allowed to provide aria-labelledby API so I think the change is fine.

**Pull request closed without merging**